sdk/core/Cargo.toml

```diff
@@ -35,9 +35,9 @@ oauth2 = "=4.0.0-alpha.3"
 reqwest = "0.11"
 paste = "1.0"
 md5 = "0.7"
+tokio = "1.0"
```

Comment:
Tokio does not support WebAssembly. I'll probably look at seeing what it takes to get core and the services to target wasm soon.

Reply:
[http://www.nooooooooooooooo.com/](http://www.nooooooooooooooo.com/)

I'll sketch a thread sleep and remove tokio.

Reply:
Done!